<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDiagramSettings</class>
 <widget class="QDialog" name="DialogDiagramSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки диаграммы</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Название диаграммы</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditTitleText"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButtonTitleFont">
         <property name="text">
          <string>Шрифт  ...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="pushButtonTitleTextColor">
         <property name="text">
          <string>Цвет текста...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Ось X(Нижняя)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditLeftAxisLabel"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButtonBottomAxisFont">
         <property name="text">
          <string>Шрифт ...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="pushButtonBottomAxisLabelTextColor">
         <property name="text">
          <string>Цвет текста...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Ось Y(Левая)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditBottomAxisLabel"/>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButtonLeftAxisFont">
         <property name="text">
          <string>Шрифт ...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="pushButtonLeftAxisLabelTextColor">
         <property name="text">
          <string>Цвет текста...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
